charts/data-dashboard-backend/values.yaml

```diff
@@ -0,0 +1,118 @@
+# Default values for data-dashboard-backend.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# -- Number of replicas to deploy
+replicaCount: 2
+
+image:
+  # -- docker image repository
+  repository: ghcr.io/thehyve/radar-data-dashboard-backend
+  # -- image pull policy
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: v0.1.3-alpha
+
+# -- Docker registry secret names as an array
+imagePullSecrets: []
+
+# -- String to partially override fullname template with a string (will prepend the release name)
+nameOverride: ""
+# -- String to fully override fullname template with a string
+fullnameOverride: ""
+
+# -- Configure pod's Security Context
+podSecurityContext: {}
+  # fsGroup: 2000
+
+# -- Configure container's Security Context
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  # -- Kubernetes Service type
+  type: ClusterIP
+  # -- data-dashboard-backend port
+  port: 9000
+
+ingress:
+  # -- Enable ingress controller resource
+  enabled: true
+  # -- Ingress class name
+  className: ""
```

Comment:
I think we should use `className` in recent version of Kubernetes instead of the annotation.

Reply:
Done.